lightly/transforms/image_grid_transform.py

```diff
@@ -0,0 +1,48 @@
+from torch import Tensor
+from PIL.Image import Image
+from typing import List, Tuple, Union
+import torchvision.transforms as T
+
+
+class ImageGridTransform:
+    """Transforms an image into multiple views.
+
+    Args:
+        transforms:
+            A sequence of transforms. Every transform creates a new view and its
+            correspondent local grid. Used for VICRegL. This argument in particular is
+            a list of tuples comprehending the cropping and the corrisponding grid
+            transformation and the non geometrical composition of transforms.
```

Comment:
Could you add some more info on the expected input? From VIGRegLTransform I gathered that the input should be `[(image_grid_transform, view_transform), (image_grid_transform, view_transform), ...]`